roles/container_runtime/tasks/package_crio.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Fail if Atomic Host since this is an rpm request
+  fail: msg='Cannot use CRI-O as a package on Atomic Host'
+  when:
+    - openshift_is_atomic | bool
+
+- include_tasks: common/pre.yml
+
+- name: Check that overlay is in the kernel
+  shell: lsmod | grep overlay
+  register: l_has_overlay_in_kernel
+  ignore_errors: yes
+  failed_when: false
+
+- when: l_has_overlay_in_kernel.rc != 0
+  block:
+
+    - name: Add overlay to modprobe.d
+      template:
+        dest: /etc/modules-load.d/overlay.conf
+        src: overlay.conf.j2
+        backup: yes
+
+    - name: Manually modprobe overlay into the kernel
+      command: modprobe overlay
+
+    - name: Enable and start systemd-modules-load
+      service:
+        name: systemd-modules-load
+        enabled: yes
+        state: restarted
+
+- name: Install cri-o
```

Comment:
If we don't have crio excluders, we need to enforce a specific version of crio here; I believe crio tracks 1-1 with kube.

Reply:
That's correct. I'll see what I can do.

Reply:
Is there a variable already set for kubernetes version that you are aware of?

Reply:
We discussed and can regroup on this later.